logio/main.go

```diff
@@ -0,0 +1,169 @@
+package logio
```

Comment:
Instead of naming this `logio`, I think that we should go ahead and name it something more generic, since this is the component that will supervise processes. Maybe we should use something that conveys the idea of "supervisor", "proxy" or "wrapper" - foreman? bigbrother? cocoon? chrysalis? saran 😆?

Reply:
Yeah, definitely needs a better name. I was thinking about just calling it `supervise`. That work?